Zend/zend_portability.h

```diff
@@ -228,6 +228,12 @@ char *alloca();
 # define ZEND_ATTRIBUTE_ALLOC_SIZE2(X,Y)
 #endif
 
+#if ZEND_GCC_VERSION >= 3000
```

Comment:
```suggestion
#if ZEND_GCC_VERSION >= 3000 || __has_attribute(const)
```
Analogous to the other blocks. Or is there a reason not to add this? It shouldn't matter for `Clang` as `ZEND_GCC_VERSION` is set to `4002` there but maybe for other compilers? I'm not sure.

Reply:
because no real use since this attribute exists since a long time 

https://gcc.gnu.org/onlinedocs/gcc-3.0.4/gcc/Function-Attributes.html

Reply:
I'm not sure we officially support anything other than gcc, clang and mscv anyway, so I guess this is fine.